ASTProcessor/src/test/java/edu/usb/argos/ASTProcessor/visitor/classes/JavaClassMemberCollectorTest.java

```diff
@@ -0,0 +1,176 @@
+package edu.usb.argos.ASTProcessor.visitor.classes;
+
+import edu.usb.argos.ASTProcessor.antlr.JavaLexer;
+import edu.usb.argos.ASTProcessor.antlr.JavaParser;
+import edu.usb.argos.ASTProcessor.visitor.core.entities.classes.ConstructorInformation;
+import edu.usb.argos.ASTProcessor.visitor.core.entities.method.AttributeInformation;
+import edu.usb.argos.ASTProcessor.visitor.core.entities.method.MethodInformation;
+import edu.usb.argos.ASTProcessor.visitor.core.services.collectors.classes.JavaClassMemberCollector;
+import edu.usb.argos.ASTProcessor.visitor.infraestructure.antlr.visitors.classes.JavaConstructorVisitor;
+import edu.usb.argos.ASTProcessor.visitor.infraestructure.antlr.visitors.method.JavaAttributeVisitor;
+import edu.usb.argos.ASTProcessor.visitor.infraestructure.antlr.visitors.method.JavaMethodVisitor;
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.ArrayList;
+
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+public class JavaClassMemberCollectorTest {
+    @Mock
+    private JavaMethodVisitor methodVisitor;
+    @Mock
+    private JavaAttributeVisitor attributeVisitor;
+    @Mock
+    private JavaConstructorVisitor constructorVisitor;
+
+    private JavaClassMemberCollector memberCollector;
+    private JavaParser.CompilationUnitContext compilationUnit;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+
+        String testClass =
+                "public class TestClass {\n" +
+                        "    private String field1;\n" +
+                        "    public Integer field2;\n" +
+                        "    public TestClass() {}\n" +
+                        "    public TestClass(String field1) { this.field1 = field1; }\n" +
+                        "    public void method1() {}\n" +
+                        "    private String method2() { return \"\"; }\n" +
+                        "}";
+
+        CharStream input = CharStreams.fromString(testClass);
+        JavaLexer lexer = new JavaLexer(input);
+        CommonTokenStream tokens = new CommonTokenStream(lexer);
+        JavaParser parser = new JavaParser(tokens);
+        compilationUnit = parser.compilationUnit();
+
+        memberCollector = new JavaClassMemberCollector(methodVisitor, attributeVisitor, constructorVisitor);
+    }
+
+    @Test
+    void getClassMethods_ShouldReturnCorrectMethods() {
```

Comment:
Use CamelCase for test name, don't snake test and the same with the rest

Reply:
Done